tck/pom.xml

```diff
@@ -92,11 +89,13 @@
         </dependency>
 
         <dependency>
-            <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-sdk-metrics</artifactId>
-            <version>1.39.0</version>
```

Comment:
Is this dependency really not needed?

I'm fairly sure it's not a transitive dependency of `microprofile-telemetry-api`, but I guess it may be a dependency of `opentelemetry-sdk-extension-autoconfigure`?

Reply:
The tests compile okay without it.

Reply:
Looks like we get `opentelemetry-sdk-extension-autoconfigure` -> `opentelemetry-sdk` -> `opentelemetry-sdk-metrics` :+1: